src/jit/gentree.h

```diff
@@ -2399,12 +2399,23 @@ struct ReturnTypeDesc
 public:
     ReturnTypeDesc()
     {
+#ifdef _TARGET_X86_
+        m_regType0 = TYP_INT;
+        m_regType1 = TYP_INT;
+
+#ifdef DEBUG
+        m_inited = true;
+#endif
+
+#else
```

Comment:
@sivarv suggested I change it this way, and I didn't realize that we were planning on using ReturnTypeDesc for all calls. I can change it back to using Initialize, but that causes some issues when we morph treenodes into helper calls (though Siva and I discussed that we may need a Reinitialize function for those cases).